test/jdk/javax/swing/text/AbstractDocument/StringContentPositionTest.java

```diff
@@ -33,9 +33,9 @@
 public class StringContentPositionTest {
```

Comment:
I believe @bug is not added because this is not strictly a regression test for this issue as I see it does not fail even without the fix?
Since `System.gc` is used, can we try restricting the heap size in commandline to see if it will fail without the fix and pass with it?